### ISSUE TITLE
Installation is So Complicated

### ISSUE THREAD

**Issue by tomanye** [frapid/frapid#39]
The architecture of design is not Easy to launch and debug  like other asp.net opensources such us Nopcommerce.

**Comment by allanregistos** [comment]
Indeed. I was lead to frapid because of mixerp. Mixerp itself is confusing. I have spent hours of trying to follow the installation procedure to no avail.
